nucypher/blockchain/eth/constants.py

```diff
@@ -17,7 +17,7 @@
 """Nucypher Token and Miner constants."""
 
 ONE_YEAR_IN_SECONDS = 31540000
-NUCYPHER_GAS_LIMIT = 6000000  # TODO: move elsewhere?
+NUCYPHER_GAS_LIMIT = 6500000  # TODO: move elsewhere?
```

Comment:
Still growing!  @szotov - In your opinion are we still within "safe" limits here?  Do you think we need to take caution when deploying to transact across multiple blocks instead of all at once?

Reply:
Right now we are in safe limit, because real gas usage around 4 millions of gas, but buffer correction and gas adjustment in tests in py-evm leads to growing this limit.

Reply:
Does it change when there is a preallocation list of addresses?

Reply:
> Do you think we need to take caution when deploying to transact across multiple blocks instead of all at once?

Perhaps this is a good Idea to build into deployment?

Reply:
Sorry, missed question before
I think we already have this in some way - wait for transaction after deployment plus local gas limit if I understand correctly